driver/longhorn

```diff
@@ -68,6 +68,10 @@ dealExistedVolume() {
     local jsonParams=$1
     local volumeName=$2
     local OUT=$(curl -s --connect-timeout 1 http://${LONGHORN_SVC}/v1/volumes/${volumeName})
+    if [ "$OUT" == "" ]; then
```

Comment:
I just realize that `OUT` probably cannot be "" otherwise curl will return an error.